screenshot/src/index.js

```diff
@@ -35,13 +52,32 @@ function fitText() {
   })
 }
 
+function updateName(template, firstname, lastname) {
+  return template
+    .replace('##FIRSTNAME##', firstname)
+    .replace('##LASTNAME##', lastname)
+}
+
+function generatePNG() {}
+
 async function main() {
   const template = await fetchTemplate()
-  await appendTemplate(template)
+  await appendTemplate(updateName(template, 'Kevin', 'Simper'))
   fitText()
-  setTimeout(() => {
-    generateCanvas()
+  setTimeout(async () => {
+    const canvas = await generateCanvas()
+    canvas.toBlob(blob => {
+      let reader = new FileReader()
+      reader.onloadend = async function() {
+        console.log(reader.result)
+        console.log(device)
+        const result = await rotate(Buffer.from(reader.result))
+        const rasterResult = raster(result)
+        // device.transferOut(2, rasterResult)
```

Comment:
This is for force printing or just debugging purposes?

Reply:
This is an accident, I have to build in a test mode 👍 good catch